index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <form id="form" onsubmit="formHandler(event)">
+        <label for="firstName">First Name:</label>
+        <input type="text" id="firstName" name="firstName" required>
+        <label for="lasttName">Last Name:</label>
+        <input type="text" id="lastName" name="lastName" required>
+        <label for="age">Age:</label>
+        <input onkeyup="validateInput('age')" type="number" id="age" name="age" required>
+        <label for="phoneNum">Phone number:</label>
+        <input onkeyup="validateInput('phoneNum')" type="number" id="phoneNum" name="phoneNum">
+        <label for="email">Email:</label>
+        <input type="email" id="email" name="email">
+        <label for="gender">Gender</label>
+        <div class="gender">
+            <label for="male">Male</label>
+            <input type="radio" id="male" name="gender">
+            <label for="female">Female</label>
+            <input type="radio" id="female" name="gender">
+        </div>
+        <p id="errorMsg"></p>
+        <button id="submit" value="submit">Submit</button>
```

Comment:
@Arzoo-25 Can you change this `button` id `submit` to `submitButton`  for better understanding?